deploy-commands.js

```diff
@@ -1,5 +1,5 @@
 const { REST, Routes } = require('discord.js');
-const { clientId, guildId, token } = require('./secrets.json');
```

Comment:
This should show secrets.json

Reply:
Use soft linking to link secrets.json to secrets_5XB.json on development and secrets_W5AC.json on production

Reply:
Missed that one